dev/PackageManager/API/M.W.M.D.AddPackageOptions.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <IsWindowsVersion.h>
+
+#include "M.W.M.D.AddPackageOptions.h"
+#include "Microsoft.Windows.Management.Deployment.AddPackageOptions.g.cpp"
+
+namespace winrt::Microsoft::Windows::Management::Deployment::implementation
+{
+    winrt::Microsoft::Windows::Management::Deployment::PackageVolume AddPackageOptions::TargetVolume()
+    {
+        return m_targetVolume;
```

Comment:
There's a version of WIL that lets you put all these simple properties right into the header file. See https://github.com/microsoft/wil/blob/master/include/wil/cppwinrt_authoring.h for how to use it.

Reply:
I'm not following. I see in cppwinrt_authoring.h support for observer/change notifications and some things related to `single_threaded_property` but just a lot of implementation. No clear code (or docs or examples) doing what you describe. What line#s are you referring to? Or link to WIL docs on the subject?